spec/hbw/features/select_table_spec.rb

```diff
@@ -34,4 +38,101 @@
 
     expect_widget_presence
   end
+
+  describe 'will be submitted if' do
+    before(:each) do
+      click_on 'Orders'
+      expect(page).to have_content 'Orders list'
+      expect_widget_presence
+    end
+
+    scenario 'empty nullable multi = false' do
+      click_and_wait 'ORD-18'
+
+      expect(page).to have_selector "button[type='submit']"
+
+      click_and_wait 'Submit'
+
+      expect(page).not_to have_content 'Field does not match regex'
+      expect(page).not_to have_content 'Field is required'
+
+      expect_widget_presence
+    end
+
+    scenario 'empty nullable multi = true' do
+      click_and_wait 'ORD-19'
+
+      expect(page).to have_selector "button[type='submit']"
+
+      click_and_wait 'Submit'
+
+      expect(page).not_to have_content 'Field does not match regex'
+      expect(page).not_to have_content 'Field is required'
+
+      expect_widget_presence
+    end
+
+    scenario 'filled required multi = false' do
+      click_and_wait 'ORD-20'
+
+      click_td_by_text 'Some name'
+
+      expect(page).to have_selector "button[type='submit']"
+
+      click_and_wait 'Submit'
+
+      expect(page).not_to have_content 'Field does not match regex'
+      expect(page).not_to have_content 'Field is required'
+
+      expect_widget_presence
+    end
+
+    scenario 'filled required multi = true' do
```

Comment:
Okay.. Field if required, you don't set any value, but you can submit. I don't get the idea of this test, sorry.

Reply:
Actually I set value twice on lines 93 & 94.

Reply:
There is no chance to understand this, rename method ```click_td_by_text```, please